README.md

```diff
@@ -3,18 +3,47 @@ A list of resources for data science work on climate change
 
 ## Data Sets
 
+### Global
+* [ICE Embodied Carbon Database](https://circularecology.com/embodied-carbon-footprint-database.html)
+* [Carbon in food downloadable csv](https://ourworldindata.org/food-choice-vs-eating-local)
+* [EPW Weather files (for Ladybug on Grasshopper, but adaptable)](https://www.ladybug.tools/epwmap/)
+* [Renewable Ninja - front end for NASA dataset](https://www.renewables.ninja/)
+* [One of many NASA datasets. This one covers precipitation](https://disc.gsfc.nasa.gov/datasets/GPM_3IMERGHH_06/summary?keywords=IMERG)
+
+* [ Keen cyclist route heatmap](https://www.strava.com/heatmap)
+* [Havard atlas of economic complexity downloads](https://atlas.cid.harvard.edu/data-downloads)
```

Comment:
This is an economic data set more than one for climate change. Yes, there is a link, but the intent of this list is for _explictly_ climate change related resources.